llvm/lib/Target/RISCV/RISCVProcessors.td

```diff
@@ -102,7 +102,12 @@ def GENERIC_RV64 : RISCVProcessorModel<"generic-rv64",
                    GenericTuneInfo;
 // Support generic for compatibility with other targets. The triple will be used
 // to change to the appropriate rv32/rv64 version.
-def GENERIC : RISCVTuneProcessorModel<"generic", NoSchedModel>, GenericTuneInfo;
+// `generic` is expected to target in-order application processors designed for
+// general-purpose computing.
+def GENERIC : RISCVTuneProcessorModel<"generic", SpacemitX60Model>,
```

Comment:
Doesn't RISCVSubtarget.cpp always remap "generic" to "generic-rv32" or "generic-rv64"?

Please see description here edd6632127975c61181dc8003c1271d4d86255b1

Reply:
We've been saying -mtune=generic, but I think what we actually mean was no -mtune provided. Meaning default behavior.

Reply:
> no -mtune provided. Meaning default behavior.

fair enough, if that's the case I guess we have to bite the bullet and update all the codegen tests.